runtime/kusama/src/lib.rs

```diff
@@ -640,16 +632,11 @@ parameter_types! {
 	pub const MaxPeerDataEncodingSize: u32 = 1_000;
 }
 
-type ApproveOrigin = EitherOfDiverse<
-	EnsureRoot<AccountId>,
-	pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 3, 5>,
->;
-
 impl pallet_treasury::Config for Runtime {
 	type PalletId = TreasuryPalletId;
 	type Currency = Balances;
-	type ApproveOrigin = ApproveOrigin;
-	type RejectOrigin = MoreThanHalfCouncil;
+	type ApproveOrigin = EnsureRoot<AccountId>;
```

Comment:
This one in particular won't be used any more since we have `SpendOrigin` to take over from it. Others are likely for components which are also defunct now. If there's one in particular you're concerned about, best to specify it explicitly.